BAZProject/MainMovie/MainMovieRouter.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  MainMovieRouter.swift
+//  BAZProject
+//
+//  Created 1030364 on 18/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class MainMovieRouter: MainMovieWireframeProtocol {
```

Comment:
Considera marcar como final todas las clases para el próximo PR
```suggestion
final class MainMovieRouter: MainMovieWireframeProtocol {
```